src/applications/mhv/landing-page/selectors/isLandingPageEnabledForUser.js

```diff
@@ -19,14 +18,12 @@ import FEATURE_FLAG_NAMES from '~/platform/utilities/feature-toggles/featureFlag
  * current user. Returns false, otherwise.
  */
 export const isLandingPageEnabledForUser = state => {
+  let isEnabled = false;
   const loggedIn = isLoggedIn(state);
-  if (!loggedIn) return false;
-  const mhvlpFeatureToggle = FEATURE_FLAG_NAMES.mhvLandingPageEnabled;
-  const featureToggleEnabled = toggleValues(state)[mhvlpFeatureToggle];
-  if (!featureToggleEnabled) return false;
-  // selectPatientFacilites alters the facilities array using map. You _must_
-  // use this selector. DO NOT traverse state. e.g. - state.user.profile.facilities
-  const facilities = selectPatientFacilities(state) || [];
-  const hasFacilities = facilities.length > 0;
-  return loggedIn && featureToggleEnabled && hasFacilities;
+  if (loggedIn) {
```

Comment:
Yes, it is possible, but having the isEnabled variable makes it easy to use the browser's debugger to force the function to return the value we need since we cannot control the facilities a user has.

Reply:
The value is returned directly in the refactored code. The way you have it appears to be a benefit only for debugging, why would we want this in production @carlosfelix2 ?

Reply:
Locally it is not possible to login and show the landing page, so having this allows us to view the landing page without having to change code. Is there another way to be able to enable the landing page locally?

Reply:
I understand having it locally but I'm operating under the assumption that we would want to remove local debugging for production. I'm not really sure, I need to do more R&D.

Reply:
I would like to leave as it to only have one return in the function. This is not debugging code, just style, but makes it easier to debug too.

Reply:
I'm not sure I follow, you want to leave it to only have one return in the function? The refactor only has one return, and the original only has one return so I am not sure what you mean. The style is good for debugging, but not something we need in production.

Reply:
> Locally it is not possible to login and show the landing page, so having this allows us to view the landing page without having to change code.

I might be able to help with this. I run `localStorage.setItem('hasSession', true)` in the browser console and run `yarn mock-api --responses src/applications/mhv/landing-page/mocks/api` (in addition to local webserver, etc.) to view the landing page locally. 

I think we could update the mocks to include a user that doesn't have facilities, though at the moment I am commenting out the mocked facilities in `src/applications/mhv/landing-page/mocks/api/user/index.js` so ignore me if that's not useful here and now

Reply:
Using the mock-api is really an ideal experience when developing. It's so much better than trying to find a test user account with the necessary data to test your changes. [This document from the Authenticated Experience team](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/engineering-docs/frontend-getting-started.md#useful-terminal-scripts) should be incorporated into the [Platform documentation](https://depo-platform-documentation.scrollhelp.site/).

Reply:
How about this new change? I'll have to look into the mocks to see how that works.